OrbitCore/LinuxUtils.cpp

```diff
@@ -107,7 +108,9 @@ void ListModules(pid_t pid,
 
     // tokens[4] is the inode column. If inode equals 0, then the memory is not
     // mapped to a file (might be heap, stack or something else)
-    if (tokens.size() != 6 || tokens[4] == "0") continue;
+    if (tokens.size() != 6) continue;
```

Comment:
So this excludes anonymous maps, right? Then the comment above refers not to this line but the next one (and this one should have its own comment).